src/payment/payment-strategy-action-creator.spec.ts

```diff
@@ -455,6 +455,26 @@ describe('PaymentStrategyActionCreator', () => {
                 expect(action.payload).toBeInstanceOf(OrderFinalizationNotRequiredError);
             }
         });
+
+        it('returns rejected promise if payment method is no longer available', async () => {
```

Comment:
`returns rejected promise if payment method referenced in order no longer exists` ?